rocketmq-console/src/main/java/org/apache/rocketmq/console/aspect/admin/MQAdminAspect.java

```diff
@@ -54,11 +61,12 @@ public Object aroundMQAdminMethod(ProceedingJoinPoint joinPoint) throws Throwabl
             MethodSignature signature = (MethodSignature)joinPoint.getSignature();
             Method method = signature.getMethod();
             MultiMQAdminCmdMethod multiMQAdminCmdMethod = method.getAnnotation(MultiMQAdminCmdMethod.class);
+            String accessKey = rmqConfigure.getAccessKey();
```

Comment:
Maybe,users can set null or emtpy String from config file.So,you should judge whether they is empty or not?If users open the aclenable flag and the accessKey or secretKey value is null,you can print the warning or erroring informations in the log.And,here you must think of another condition that users may not want to use the acl feature!